src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -129,6 +129,20 @@ public long GetMinimalEtagForReplication()
             }
 
             var replicationNodes = Destinations?.ToList();
+            foreach (var rehabNode in Database.ReadDatabaseRecord().Topology.Rehabs)
```

Comment:
I think it is better to be on the safe side and include all nodes here

Reply:
also `ReadDatabaseRecord` might be expensive, while here only the topology is required. (if possible, consider to get it while you open the read tx above)

Reply:
done